dll/win32/shell32/lang/ro-RO.rc

```diff
@@ -842,24 +842,24 @@ BEGIN
 
     /* message box strings */
     IDS_RESTART_TITLE "Repornire"
-    IDS_RESTART_PROMPT "Doriți să reporniți sistemul?"
+    IDS_RESTART_PROMPT "Reporniți sistemul?"
     IDS_SHUTDOWN_TITLE "Închidere"
-    IDS_SHUTDOWN_PROMPT "Doriți să închideți calculatorul?"
+    IDS_SHUTDOWN_PROMPT "Închideți calculatorul?"
 
     /* Format Dialog Strings */
     IDS_FORMAT_TITLE "Formatare disc local"
-    IDS_FORMAT_WARNING "AVERTISMENT: Formatarea va șterge TOATE datele de pe acest disc.\nConfirmați pentru a formata, anulați pentru a ieși."
-    IDS_FORMAT_COMPLETE "Formatare completă."
+    IDS_FORMAT_WARNING "AVERTISMENT: Formatarea va şterge TOATE datele de pe acest disc.\nPentru a formata discul, faceţi clic pe OK. Pentru a ieşi, faceţi clic pe REVOCARE."
+    IDS_FORMAT_COMPLETE "Formatare terminată."
 
     /* Warning format system drive dialog strings */
-    IDS_NO_FORMAT_TITLE "Nu a putut fi formatat volumul"
-    IDS_NO_FORMAT "Nu a putut fi formatat acest volum! Conține fișiere de sistem importante care să facă ReactOS să meargă."
+    IDS_NO_FORMAT_TITLE "Imposibil de formatat volumul"
+    IDS_NO_FORMAT "Imposibil de formatat acest volum! Conține fișiere de sistem importante care fac ReactOS să funcționeze."
 
     /* Run File dialog */
-    IDS_RUNDLG_ERROR "„Executare fișier” nu a putut fi deschis (eroare internă)"
-    IDS_RUNDLG_BROWSE_ERROR "„Specificare fișiere” nu a putut fi deschisă (eroare internă)"
+    IDS_RUNDLG_ERROR "Imposibil de mutat acest folder (eroare internă)."
+    IDS_RUNDLG_BROWSE_ERROR "Imposibil de mutat acest fișier (eroare internă)"
     IDS_RUNDLG_BROWSE_CAPTION "Specificare fișiere"
-    IDS_RUNDLG_BROWSE_FILTER "Fișiere executabile (*.exe)\0*.exe\0Orice fișier (*.*)\0*.*\0"
+    IDS_RUNDLG_BROWSE_FILTER "Programe#*.exe\0*.exe\0Toate fişierele (*.*)\0*.*\0"
```

Comment:
```suggestion
    IDS_RUNDLG_BROWSE_FILTER "Programe (*.exe)\0*.exe\0Toate fişier (*.*)\0*.*\0"
```
That hashtag is really intended? I do assume that was an oopsie, because at another line you did:
IDS_OPEN_WITH_FILTER "Programe (*.exe)\0*.exe\0Toate fișierele (*.*)\0*.*\0"
So this is very inconsistent after your change now. Also with the removed () around (*.exe) while keeping it at (*.*)
That makes no sense imho.

Reply:
Yes, you are right. I accidentally added the hashtag. I wanted to add space.